src/services/match.js

```diff
@@ -340,6 +340,27 @@ const removeExistingMatch = async(req, res, next) =>
     }
 }
 
+const cancelSendRequest = async (req, res, next) => 
+{
+    try 
+    {
+        const matchId = req.params.matchId;
+        const match = await Match.findById(matchId);
+
+        if (!match) return res.status(404);
+        
+        removeMatchHelper(match);
+
+        return res.status(200).json({});
```

Comment:
2xx is considered a success response for api requests, but what are we succeeding here? It would be good to return a JSON with message that match request successfully cancelled/removed.